patches/099-sqlite_api_3_26_0.patch

```diff
@@ -216,3 +1161,36 @@ index fa7d9daadd0739b6ea11189b238dbc6c59f59287..fc233a1205a0611ed017cdb615fc5e01
  }
  
  }  // namespace blink
+diff --git a/third_party/sqlite/BUILD.gn b/third_party/sqlite/BUILD.gn
+index 4f31436ef8e5..d23cf877dd40 100644
+--- a/third_party/sqlite/BUILD.gn
++++ b/third_party/sqlite/BUILD.gn
```

Comment:
This patch is not required, these changes to `//third_party/sqlite` are already present in `098-sqlite_upgrade_from_3_20_0_to_3_26_0.patch`.

Reply:
Removed from `099` in 88a5ee3